orchestrator/utils/get_updated_fields.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019-2020 SURF.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+
+def get_dict_updates(obj1: dict, obj2: dict, delta: bool = False) -> dict:
```

Comment:
Mixed use of dict and Dict in the orchestrator-core codebase

Reply:
yea, 3.9 -_-